ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -127,6 +127,14 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
   }
 
+  public boolean onKeyDown(int keyCode, KeyEvent event) {
+    if (keyCode == KeyEvent.KEYCODE_MEDIA_FAST_FORWARD) {
+      event.startTracking();
```

Comment:
should we only do this when `getReactNativeHost().getUseDeveloperSupport()` is true?

Reply:
Hm, this check is currently done in `onKeyLongPress` (https://github.com/facebook/react-native/pull/16500/files#diff-f3ee27db155d4af4368a3ff1e60a765cR156).
Do you think it should be moved to here, or do you think it's worth keeping it in both places?

Reply:
add it in both, we don't need to `startTracking` unless we are in dev mode